rest_lights.cpp

```diff
@@ -624,31 +620,36 @@ int DeRestPluginPrivate::setLightState(const ApiRequest &req, ApiResponse &rsp)
             if (map[param].type() == QVariant::Double)
             {
                 const int briInc = map[param].toInt(&ok);
-                if (ok && briInc >= -254 && briInc <= 254)
+                if (ok && briInc >= -0xFF && briInc <= 0xFF)
                 {
                     valueOk = true;
                     hasBriInc = true;
-                    targetBriInc = briInc;
+                    targetBriInc = briInc < -0xFE ? -0xFE : briInc > 0xFE ? 0xFE : briInc;
                 }
             }
         }
         else if (param == "xy"  && taskRef.lightNode->item(RStateX) && taskRef.lightNode->item(RStateY) &&
                  taskRef.lightNode->modelId() != QLatin1String("FLS-PP"))
         {
-            // @manup: is check for FLS-PP needed, or is this already handled by check for state.xy?
+            // @manup: is check for FLS-PP needed, or is this already handled by check for state.x and state.y?

```

Comment:
Hmm good question: The check refers to the very old "FLS-PP" wireless ballast from 2012 which didn't support xy color mode there are less than 200 deployed but still active. When they are configured we'd need to strip the xy ResourceItems (the ballast wrongly claims it supports this color mode).

The newer ones with model id "FLS-PP3" support xy and don't fall into this check.

Reply:
You make devices of too high quality, now you never get rid of old stuff ;-)

Another argument for capabilities...